### ISSUE TITLE
Consider SVG asciinema in readme

### ISSUE THREAD

**Issue by thcyron** [hetznercloud/cli#132]
Would be nice because it would play in the readme. No need to go the the asciinema website.

https://github.com/marionebl/svg-term-cli

**Comment by philippgille** [comment]
I liked the idea and tried it out.

Command to generate the SVG (I don't want to install anything):

`docker run --rm -v /tmp/asciinema-svg:/tmp/asciinema-svg node:alpine sh -c "npm install -g svg-term-cli && svg-term --cast=157991 --out /tmp/asciinema-svg/hcloud.svg"`

Unfortunately GitHub doesn't support uploading SVG files in issues, nor do the image hosters that I tried. But here's the SVG in a ZIP archive: 
[hcloud-asciinema-svg.zip](https://github.com/hetznercloud/cli/files/2704221/hcloud-asciinema-svg.zip)

There are two disadvantages:

1) The asciinema video allows you to pause the video at any time and then easily copy and paste the commands that are on the screen in that moment. The SVG can neither be paused, nor can any commands be easily copied. At least where I tried to view the SVG, the selected text became unselected with each change in the SVG (like another character being typed in the video and appearing on the screen).
2) During the asciinema video the Chrome process has a CPU usage of less than 1%. When opening the SVG from a local file, the CPU usage is at 8%. With an AMD Ryzen 5 1600X CPU (6 x 3.6 GHz).

So I think overall it's better to keep linking to asciinema.
